core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -218,7 +280,49 @@ describe('Overview', () => {
             browser.sleep(2000);
             jazzServices_po.getServiceFromAsset().click();
       });
-      
+
+      it('Verify METRICS' , () => {
```

Comment:
can you give different description to the tests. I cant identify the diff
also do you think the sleep is too much (30 sec, 45 secs and then more is added).

Reply:
For Metrics- written everything in a block so that it will execute as single test case. Metrics Swagger changes will take time .For Adjusting that changes variations has given to other test cases